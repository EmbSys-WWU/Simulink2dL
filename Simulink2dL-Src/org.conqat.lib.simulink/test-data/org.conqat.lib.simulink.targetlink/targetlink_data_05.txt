struct('revisionNumber',9,'description','','stepfunctionname','F_RampDynamic_Part2','stepfunctionclass','GLOBAL_FCN','forcerestartfunction',0,'restartfunctionname','RESTART_$S_$B','restartfunctionclass','default','forceinitfunction',0,'initfunctionname','INIT_F_RampDynamic_Part2','initfunctionclass','default','forcetermfunction',0,'termfunctionname','TERM_$S_$B','termfunctionclass','default','subsystemid','FB24b','forcereuse',1,'linkfile',0,'functionarglist',{{}},'unscaledarguments',1,'scalingpropagation',{{['  % mark all inports and outports as scaling invariant '];['    Ret_In(2).Invariant = 1;'];['    Ret_In(3).Invariant = 1;'];['    Ret_Out(1).Invariant = 1;'];['    % calulate the scaling and the ranges of the output'];['    Ret_Out(1).Lsb    =SII_In(2).Lsb;'];['    Ret_Out(1).Offset = SII_In(2).Offset;'];['    Ret_Out(1).Min = SII_In(2).Min;'];['    Ret_Out(1).Max = SII_In(2).Max;']}},'incrcodegen',0,'ownedmodules',{{}},'destination','','addtestframe',0,'simmode',1,'codecomment','','codefile','F_RampDynamic_Part2','codecoveragelevel',0)