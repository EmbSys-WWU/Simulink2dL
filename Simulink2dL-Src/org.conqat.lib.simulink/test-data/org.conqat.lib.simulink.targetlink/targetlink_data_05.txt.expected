struct (
  'addtestframe' -> '0'
  'codecomment' -> ''
  'codecoveragelevel' -> '0'
  'codefile' -> 'F_RampDynamic_Part2'
  'description' -> ''
  'destination' -> ''
  'forceinitfunction' -> '0'
  'forcerestartfunction' -> '0'
  'forcereuse' -> '1'
  'forcetermfunction' -> '0'
  'functionarglist' -> '{{}}'
  'incrcodegen' -> '0'
  'initfunctionclass' -> 'default'
  'initfunctionname' -> 'INIT_F_RampDynamic_Part2'
  'linkfile' -> '0'
  'ownedmodules' -> '{{}}'
  'restartfunctionclass' -> 'default'
  'restartfunctionname' -> 'RESTART_$S_$B'
  'revisionNumber' -> '9'
  'scalingpropagation' -> '{{[  % mark all inports and outports as scaling invariant ];[    Ret_In(2).Invariant = 1;];[    Ret_In(3).Invariant = 1;];[    Ret_Out(1).Invariant = 1;];[    % calulate the scaling and the ranges of the output];[    Ret_Out(1).Lsb    =SII_In(2).Lsb;];[    Ret_Out(1).Offset = SII_In(2).Offset;];[    Ret_Out(1).Min = SII_In(2).Min;];[    Ret_Out(1).Max = SII_In(2).Max;]}}'
  'simmode' -> '1'
  'stepfunctionclass' -> 'GLOBAL_FCN'
  'stepfunctionname' -> 'F_RampDynamic_Part2'
  'subsystemid' -> 'FB24b'
  'termfunctionclass' -> 'default'
  'termfunctionname' -> 'TERM_$S_$B'
  'unscaledarguments' -> '1'
)